iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractNetworkTest.java

```diff
@@ -626,4 +626,16 @@ public void testScadaNetwork() {
             // Ignore
         }
     }
+
+    @Test
+    public void testPermanentLimit() {
+        Network network = EurostagTutorialExample1Factory.createWithFixedCurrentLimits();
+        assertEquals(ValidationLevel.STEADY_STATE_HYPOTHESIS, network.getValidationLevel());
+        ValidationException e = assertThrows(ValidationException.class, () -> network.getLine("NHV1_NHV2_1").getCurrentLimits2().orElseThrow().setPermanentLimit(Double.NaN));
+        assertTrue(e.getMessage().contains("AC line 'NHV1_NHV2_1': permanent limit must be defined if temporary limits are present"));
+        network.setMinimumAcceptableValidationLevel(ValidationLevel.EQUIPMENT);
+        network.getLine("NHV1_NHV2_1").getCurrentLimits2().orElseThrow().setPermanentLimit(Double.NaN);
+        assertTrue(Double.isNaN(network.getLine("NHV1_NHV2_1").getCurrentLimits2().orElseThrow().getPermanentLimit()));
+        assertEquals(ValidationLevel.STEADY_STATE_HYPOTHESIS, network.getValidationLevel()); // not supported through setters...
```

Comment:
I think it is important to have this.
This relies on the `ValidationUtil.checkIdentifiable(...)` method which should now consider the limits to determine the validation level of a network.

Reply:
I pushed modifications to take the limits into account when determining the validation level of the network.

I choose to consider all the `OperationalLimitsGroup`s (the selected and the unselected) since it is easier to do: when considering only the selected ones, the validation level should be checked every time the selected operationalLimitsGroup is changed for any limits holder.